.circleci/config.yml

```diff
@@ -287,9 +327,10 @@ jobs:
           keys:
             - dep-cache-{{ checksum "Gopkg.toml" }}-{{ checksum "WORKSPACE" }}
```

Comment:
remove workspace? Not sure if this should be pinned with Gopkg.lock..

Reply:
Ongoing discussion, we'll address it in separate PR.